addons/calendar/data/mail_template_data.xml

```diff
@@ -22,7 +22,12 @@
         Hello <t t-out="object.common_name or ''">Wood Corner</t>,<br/><br/>
```

Comment:
@amdi-odoo ... And for this PR: do not forget to update the Commit Message to match the content. Cheers!